packages/documentation/src/shared/snapshots/schemes.ts

```diff
@@ -14,13 +14,18 @@ export const COLOR_SCHEMES = {
 export function schemes(renderFn: (scheme: string) => TemplateResult, options: IOptions = {}) {
   const filter = options.filter || (() => true);
   const additionalSchemes = options.additionalSchemes ?? [];
+  const schemes = [...Object.values(COLOR_SCHEMES), ...additionalSchemes].filter(filter);
 
-  return html`${[...Object.values(COLOR_SCHEMES), ...additionalSchemes].filter(filter).map(
-    scheme => html` <div data-color-scheme="${scheme}">
-      <div class="p-16 palette palette-default">
-        <p>Color Scheme: ${scheme}</p>
-        ${renderFn(scheme)}
-      </div>
-    </div>`,
-  )}`;
+  return html`
+    ${schemes.map(
+      scheme => html`
+        <div data-color-scheme="${scheme}">
+          <div class="p-16 palette palette-default">
```

Comment:
```suggestion
          <div class="palette palette-default">
```

The snapshots sometimes need to be full-screen, for example the full-width tabs.
Adding an option to remove the padding would also work.